worldengine/cli/main.py

```diff
@@ -543,7 +550,10 @@ def main():
             '%s/%s_scatter.png' % (args.output_dir, world_name))    
         if args.satelite_map:
             draw_satellite_map(world,
-            '%s/%s_satellite.png' % (args.output_dir, world_name))    
+            '%s/%s_satellite.png' % (args.output_dir, world_name))
+        if args.icecaps_map:
+            draw_icecaps_map(world,
+            '%s/%s_icecaps.png' % (args.output_dir, world_name))
```

Comment:
I added an option to look at the height of the ice. Not too useful currently, but maybe it will be in the future.